src/llama-vocab.cpp

```diff
@@ -2103,6 +2104,12 @@ void llama_vocab::impl::load(llama_model_loader & ml, const LLM_KV & kv) {
             } else {
                 _set_token_attr("<mask>", LLAMA_TOKEN_ATTR_LSTRIP, true);
             }
+        } else if (_contains_any(general_arch, {"modern-bert"})) {
+            if (token_to_id.count("[MASK]") == 0) {
+                LLAMA_LOG_WARN("%s: Mask token is missing in vocab, please reconvert model!\n", __func__);
```

Comment:
```suggestion
                LLAMA_LOG_WARN("%s: Mask token not found in vocab!\n", __func__);
```
nit: This would likely not be a conversion error, but a modified vocab.

Reply:
Yeah, let me change that to not make user confused